cmd/zetaclientd/start.go

```diff
@@ -117,7 +117,12 @@ func start(_ *cobra.Command, _ []string) error {
 		}
 	}()
 
-	tss, err := GenerateTss(masterLogger, cfg, zetaBridge, peers, priKey, telemetryServer)
+	tssHistoricalList, err := zetaBridge.GetTssHistory()
+	if err != nil {
+		startLogger.Error().Err(err).Msg("GetTssHistory error")
+	}
+
+	tss, err := GenerateTss(masterLogger, cfg, zetaBridge, peers, priKey, telemetryServer, tssHistoricalList)
```

Comment:
Maybe we should explicitly set an empty array for `tssHistoricalList` if the previous method returned an error